jwst/resample/resample.py

```diff
@@ -190,8 +190,8 @@ def do_drizzle(self):
                 exposure_times['end'].append(img.meta.exposure.end_time)
 
                 # apply sky subtraction
-                if 'skybg' in img.meta._instance:
-                    img.data -= img.meta.skybg
+                if not img.meta.background.subtracted:
+                    img.data -= img.meta.background.level
```

Comment:
Do we need a check here to make sure that img.meta.background.level actually exists and/or is populated with a value? E.g. what if the skymatch step was not run upstream? If so, this attribute could be None and hence the arithmetic operator would not be too keen about that.

Reply:
It probably would be a good idea to add such a check.  I will look into it immediately.